csrc/host_ir/jit.cpp

```diff
@@ -42,15 +42,25 @@
 
 namespace nvfuser {
 
-using AllocateFunc = std::function<
-    void(const int64_t*, int64_t, const int64_t*, int64_t, at::Tensor&)>;
+using main_func_t = std::function<at::Tensor**(const at::Tensor**)>;
+constexpr std::string_view main_func_name = "main";
```

Comment:
```suggestion
constexpr std::string_view kMainFuncName = "main";
```

Reply:
Same for other constexprs. https://google.github.io/styleguide/cppguide.html#Constant_Names